pyscitt/pyscitt/client.py

```diff
@@ -15,22 +16,31 @@
 from cryptography.hazmat.primitives.serialization import load_pem_private_key
 from loguru import logger as LOG
 
-from . import crypto
 from .governance import GovernanceClient
 from .prefix_tree import PrefixTreeClient
 from .receipt import Receipt
 
 CCF_TX_ID_HEADER = "x-ms-ccf-transaction-id"
 
+
+class MemberAuthenticationMethod(ABC):
+    @property
```

Comment:
A `@property` means this method should look and feel just like a field from the outside (read-only in this case). It should be called just `key_id`, without the `get_` prefix, and should be accessed as if it were a field, not a method, ie. `member_auth_client.key_id` instead of `member_auth_client.get_key_id()` (note the lack of parenthesis).

The subclasses should either have methods that are also marked as `@property` themselves, or they should just be plain fields.

eg.
```python
class MemberAuthenticationMethod(ABC):
    @property
    @abstractmethod
    def key_id(self) -> bytes: pass

# either
class LocalKeySignClient(MemberAuthenticationMethod):
    key_id: bytes

    def __init__(self, [...]):
        self.key_id = [...]

# or
class LocalKeySignClient(MemberAuthenticationMethod):
    _key_id: bytes

    def __init__(self, [...]):
        self._key_id = [...]
 
    @property
    def key_id(self) -> bytes:
        return self._key_id
```


(It would also be fine to make this a normal method and drop the `@property` altogether.)